srcpkgs/mopidy/template

```diff
@@ -1,21 +1,21 @@
 # Template file for 'mopidy'
 pkgname=mopidy
-version=2.3.1
-revision=2
+version=3.0.1
+revision=1
 archs=noarch
 wrksrc="Mopidy-${version}"
-build_style=python2-module
+build_style=python3-module
 pycompile_module=mopidy
```

Comment:
No need anymore

Reply:
Yes !

Reply:
> Yes !

I've been told fairly recently, when updating python packages to remove them

Reply:
Removed. Where do you told things ?

Reply:
https://github.com/void-linux/void-packages/pull/17926 3rd resolved is where I was told

Reply:
Do you want me to remove + unindex python2-pykka ?

Reply:
Not sure what you are "pointing" to. I am confused because of + unindex

Reply:
I was learning about your way to mark packages as "deleted" using the removed-packages replaces. I guess the "+ unindex" actually is an automatic step executed by xbps.

So, do you prefer me to put python-pykka in the removed package replace list? It seems to be used only by mopidy atm

Reply:
Not sure how committers want to approach this currently, as I was changing them but it has been a bit of a argument ATM, so currently just leave it a mix until someone who can commit looks over this.

Reply:
Okay! Ty for your review. I mark conversation as resolved :)